Marlin/ultralcd_impl_HD44780.h

```diff
@@ -248,20 +258,11 @@ static void lcd_set_custom_characters(
   lcd.createChar(LCD_STR_BEDTEMP[0], bedTemp);
   lcd.createChar(LCD_STR_DEGREE[0], degree);
   lcd.createChar(LCD_STR_THERMOMETER[0], thermometer);
+  lcd.createChar(LCD_STR_UPLEVEL[0], uplevel);
   lcd.createChar(LCD_STR_FEEDRATE[0], feedrate);
   lcd.createChar(LCD_STR_CLOCK[0], clock);
 
   #if ENABLED(SDSUPPORT)
```

Comment:
…to an `#else` clause added to the end of this block. So:
```cpp
#if ENABLED(SDSUPPORT)
  . . .
#else
  lcd.createChar(LCD_STR_UPLEVEL[0], uplevel);
#endif
```
This simply prevents it being set twice when `LCD_PROGRESS_BAR` is enabled.